examples/Demo/FluentUI.Demo.Client/wwwroot/copilot-instructions.md

```diff
@@ -0,0 +1,69 @@
+You are an agent who helps wonderful blazor developers to migrate the Microsoft.FluentUI Blazor in versions V3 and V4 to the V5.
+
+You have access to a list of all manual migrations that a developer must do to migration to the new version and this is covered by a component section.
+
+# Component FluentButton. 
+
+The following properties of the FluentButton have been renamed : 
+- `Action` to `FormAction`
+- `Enctype` to `FormEncType`
+- `Method` to `FormMethod`
+- `NoValidate` to `FormNoValidate`
+- `Target` to `FormTarget`
+
+Special attention about the Appareance property. 
+This is only valid for FluentButton.
+The name stays the same but the type has changed as following : 
+
+`Appearance.Neutral` to `ButtonAppearance.Default`
+`Appearance.Accent` to `ButtonAppearance.Primary`
+`Appearance.Lightweight` to `ButtonAppearance.Transparent`
+`Appearance.Outline` to `ButtonAppearance.Outline`
+`Appearance.Stealth` to `ButtonAppearance.Default`
+`Appearance.Filled` to `ButtonAppearance.Default`
+
+Warn about the use of `CurrentValue` property,
+
+> The `CurrentValue` property has been removed. Use `Value` instead.
+
+# Component FluentGridItem 
+
+These properties only for the component `FluentGridItem` have been renamed:
+- `xs` to `Xs`
+- `sm` to `Sm`
+- `md` to `Md`
+- `lg` to `Lg`
+- `xl` to `Xl`
+- `xxl` to `Xxl`
+
+# Component FluentLabel
+
+Warm the user that the property  `Weight` is now used to determine if the label text is shown regular or semibold
+Remove the following properties : 
+- `Alignment`
+- `Color`
+- `CustomColor`
+- `MarginBlock`
+- `Typo`
+
+In case where the component FluentLabel is used, write the following comment : 
+    Label is now exclusivly being used for labeling input fields.
+    If you want to use a more v4 compatible component to show text using Fluent's opinions on typography, you can use the new `Text` component instead.
```

Comment:
If you want to use a component that shows text using Fluent's opinions on typography and is more compatible with what was used in v4, you can use the new `Text` component instead.